configs/olcf/config.20170915.beta2.A_WCYCL1850S.ne30_oECv3_ICG.edison

```diff
@@ -139,5 +139,8 @@ baseDirectory =  /lustre/atlas/proj-shared/cli115/milena/ACMEv0_lowres/B1850C5_n
 # Supported options are Atlantic and IndoPacific
```

Comment:
Could you change `mpasMeshName` to `oEC60to30v3`?

And then add, before line 54:
```
# provide an absolute path to put HTML in an alternative location (e.g. a web
# portal)
# htmlSubdirectory = /ccs/proj/cli115/www/USERNAME/RUNNAME
htmlSubdirectory = html
```

Reply:
done

Reply:
I think you forgot to adjust the `mpasMeshName`.

Reply:
Yes, I missed that.  I'll fix it now.